src/app/components/copy-to-clipboard/index.html

```diff
@@ -0,0 +1,40 @@
+<sky-demo-page
+  pageTitle="Copy to clipboard"
+  skyuxModule="SkyClipboardModule"
+  packageName="lib-clipboard"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The copy to clipboard component creates a button on your page that enables users to copy the content to their computer's clipboard.
+    </p>
```

Comment:
We are missing the note here about how to set up the copy to clipboard component as a dependency in order to use it. It should match the structure of the note in the code block component and pull in the information at https://developer.blackbaud.com/stache/content/clipboard#non-stache-spas.